src/watchful/__init__.py

```diff
@@ -1,4 +1,5 @@
 """Watchful python library"""
+
```

Comment:
Did formatting change this?

Reply:
Yeah. `hatch run check` was failing in the black step saying it'd reformat all of this.

Reply:
done